README.md

```diff
@@ -1,72 +1,96 @@
+| CNMF for dendrites | data_dendritic.ipynb | Developed by Eftychios Pnevmatikakis | 
+| Online CNMF (OnACID) | demo_OnACID_mesoscope.ipynb |[Giovannucci et al., 2017](https://proceedings.neurips.cc/paper_files/paper/2017/hash/4edaa105d5f53590338791951e38c3ad-Abstract.html) | 
+| Online volumetric CNMF | demo_online_3D.ipynb | Developed by Johannes Friedrich | 
+| Online CNMFE (OnACID-E) | demo_realtime_cnmfE.ipynb |[Friedrich et al. 2020](https://pubmed.ncbi.nlm.nih.gov/33507937/) | 
+| Motion correction | demo_motion_correction.ipynb | [Pnevmatikakis et al., 2017](https://pubmed.ncbi.nlm.nih.gov/28782629/) | 
+| Register cells across sessions | demo_multisession_registration.ipynb | [Pnevmatikakis et al., 2016](https://pubmed.ncbi.nlm.nih.gov/26774160/) | 
 
-There are other ways to build/use caiman, but they may get less or no support depending on how different they are.
+A comprehensive list of references, where you can find detailed discussion of the methods and their development, can be found [here](https://caiman.readthedocs.io/en/master/CaImAn_features_and_references.html#references). 
 
-More detailed docs on installation can be found [here](./docs/source/Installation.rst).
 
-After installing the software, the caimanmanager.py script (which will be put in your path on Linux and OSX) is used to unpack datafiles and demos into a directory called caiman\_data. Invoke it with the "install" argument.
+# For questions, comments, and help :question:
+If you need help using Caiman, or hit an installation snag, we recommend asking about it in the [Gitter forum](https://app.gitter.im/#/room/#agiovann_Constrained_NMF:gitter.im). If you have found a bug, we recommend searching the [issues at github](https://github.com/flatironinstitute/CaImAn/issues) and opening a new one if you can't find it there. If you have a general question about how things work or there is a feature you would like to see, feel free to come chat at Gitter or open an issue at Github. 
 
-If you are on Windows, the mechanism to put caimanmanager.py in your path may not work; in this case, you can locate it by typing python to bring up an interactive shell, and then
-```bash
->>> import sysconfig
->>> sysconfig.get_path('scripts')
-```
-You can invoke it as follows:
-```bash
-python C:/PATH/ABOVE/caimanmanager.py install
-```
+# How to contribute :hammer:
+ Caiman is an open-source project and improves because of contributions from users all over the world. If there is something about Caiman that you would like to improve, then you are qualified to contribute! We are always looking for more contributors, so please come read the [contributors page](./CONTRIBUTING.md) for more details about how. 
+
+# Videos :tv:
+These talks by Andrea Giovannucci from past Caiman workshops/events are an excellent start for newcomers. They go through NoRMCorre, CNMF(E) and VolPy.
+
+The following Open Neuroscience talk provides a good high-level introduction to Caiman:    
```

Comment:
maybe just "The following talk"

Reply:
Actually, we probably should be consistent between either saying where talks come from when we talk about them, and never doing so. So I'm no longer sure about this comment (later we mention sources of talks). Although this is potentially jarring if someone doesn't know that "Open Neuroscience" is a YT channel.

Reply:
Good point. I think maybe leaving out origin seems good do people care if it was SFN 2014? We can fix later if we want.

Reply:
I think leave the description of the talk. The open neuro one is a broad level overview, the nemonic is more detailed.

Reply:
That's what I have in my current local @kushalkolar : general description, but not of location. :)